### PR TITLE
update minio to use the official one

### PR DESCRIPTION
**Pull request by s12chung**
it's now available officially and fixes a bug
which required a fork earlier

### CODE REVIEW

File: staging/velero/templates/miniobackend.yaml [hunk 1]
```diff
@@ -123,7 +123,7 @@ spec:
       serviceAccountName: minio-operator-sa
       containers:
       - name: minio-operator
-        image: mesosphere/kubeaddons-minio-operator:v0.0.2
+        image: minio/k8s-operator:1.0.1
```
Comment (hectorj2f) [inline staging/velero/templates/miniobackend.yaml]: can you share the bug that was fixed ? So we can understand why we are changing it :).

File: staging/velero/templates/deployment.yaml [hunk 2]
```diff
@@ -120,7 +120,7 @@ spec:
       initContainers:
         {{- with .Values.minioBackend }}
         - name: initialize-velero
-          image: mesosphere/kubeaddons-addon-initializer:v0.0.4
+          image: mesosphere/kubeaddons-addon-initializer:v0.0.5
```
Comment (hectorj2f) [inline staging/velero/templates/deployment.yaml]: what did it change at this side ?

Comment (s12chung) [reply staging/velero/templates/deployment.yaml]: https://github.com/mesosphere/kubeaddons-extrasteps/pull/10 -- basically some fields in the new minio-opeator are gone now. and it updated names of some resources.

File: staging/velero/templates/miniobackend.yaml (hunk 1 above)
Comment (s12chung) [reply staging/velero/templates/miniobackend.yaml]: remember that tempfile problem we paired together on?

so i wrote this to copy the files to not make templfiles and make regular files instead:

https://github.com/s12chung/minio-operator/commit/50afb6c0b43bef4a89153c6087623bc17175b2b5#diff-3254677a7917c6c01f55212f86c57fbfL11

which points to this: https://github.com/s12chung/minio

File: staging/velero/templates/deployment.yaml (hunk 2 above)
Comment (hectorj2f) [reply staging/velero/templates/deployment.yaml]: ack

File: staging/velero/templates/miniobackend.yaml (hunk 1 above)
Comment (hectorj2f) [reply staging/velero/templates/miniobackend.yaml]: thanks, the explanation would help any reviewer to understand your changes ;)

**Pull request merged**